packages/mui-material/src/Chip/Chip.js

```diff
@@ -569,6 +571,10 @@ Chip.propTypes /* remove-proptypes */ = {
     PropTypes.func,
     PropTypes.object,
   ]),
+  /**
+   * tab order of an element
+   */
+  tabIndex: PropTypes.number,
```

Comment:
I think we don't want to document it as a prop as it is a HTML attribute. You can `@ignore` it.
```suggestion
  /**
   * @ignore
   */
  tabIndex: PropTypes.number,
```

Reply:
ahh okay got it. will update

Reply:
Updated